Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -5,399 +5,385 @@
 using Frends.FTP.DownloadFiles.Enums;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/SingleFileTransfer.cs#L1-L3
You are not using these namespaces, so they should be removed.